.github/workflows/tests.yml

```diff
@@ -30,6 +30,12 @@ jobs:
       PGPASSWORD: discourse
       USES_PARALLEL_DATABASES: ${{ matrix.build_type == 'backend' || matrix.build_type == 'system' }}
       CAPYBARA_DEFAULT_MAX_WAIT_TIME: 10
+      MINIO_RUNNER_LOG_LEVEL: DEBUG
+      MINIO_RUNNER_MINIO_ROOT_USER: minioadmin
+      MINIO_RUNNER_MINIO_ROOT_PASSWORD: minioadmin
+      MINIO_RUNNER_MINIO_DOMAIN: minio.local
+      MINIO_RUNNER_BUCKETS: "discoursetest"
+      MINIO_RUNNER_PUBLIC_BUCKETS: "discoursetest"
```

Comment:
I think these are all the defaults? Any reason to override them here?

Reply:
Ah yeah good point we don't really need these since we are setting stuff in `rails_helper`, will remove :+1: